libff/algebra/curves/bls12_377/bls12_377_pairing.cpp

```diff
@@ -282,18 +289,20 @@ void bls12_377_doubling_step_for_miller_loop(const bls12_377_Fq two_inv,
     //
     // The tangent line evaluated at the twisting point P = (x_p, y_p) is
     // computed as:
-    // (-2YZ*y_p)vw + (3*X^2 * x_p) * v^2 + bls12_377_twist * bls12_377_twist * (3*bls12_377_twist_coeff_b*Z^2 - Y^2)
+    // (-2YZ*y_p)vw + (3*X^2 * x_p) * v^2 + bls12_377_twist * bls12_377_twist *
+    // (3*bls12_377_twist_coeff_b*Z^2 - Y^2)
```

Comment:
This comment is now broken.. the "arrows" do not point to the right terms anymore

Reply:
Reformatted the comment.